File: C#/problems.cs
using System;

class Program
{
    static int Add(int a, int b) { return a + b; }
    static int Subtract(int a, int b) { return a - b; }
    static int Multiply(int a, int b) { return a * b; }
    static int Divide(int a, int b) { return b != 0 ? a / b : 0; } // Add error handling for division by zero

    static void Main()
    {
        int x = 10, y = 5;
        Console.WriteLine("Add: " + Add(x, y));
        Console.WriteLine("Subtract: " + Subtract(x, y));
        Console.WriteLine("Multiply: " + Multiply(x, y));
        Console.WriteLine("Divide: " + Divide(x, y));
    }
}

File: RUBY/problems.rb
def add(a, b) a + b end
    def subtract(a, b) a - b end
    def multiply(a, b) a * b end
    def divide(a, b) b != 0 ? a / b : nil end # Add error handling for division by zero
    
    x, y = 10, 5
    puts "Add: #{add(x, y)}"
    puts "Subtract: #{subtract(x, y)}"
    puts "Multiply: #{multiply(x, y)}"
    puts "Divide: #{divide(x, y)}"
    